src/main/java/org/broadinstitute/barclay/argparser/CommandLineProgramProperties.java

```diff
@@ -27,5 +26,6 @@
      */
     String usageExample() default "The author of this program hasn't included any example usage, please complain to them.";
```

Comment:
Would be great (future feature) to be able to emit a list of usage-lacking tools at build time.